packages/server/test/mail.integration.ts

```diff
@@ -38,4 +42,24 @@ describe('Mail Integration', () => {
         .expect(202);
     });
   });
+
+  describe('Find all mail services for a user', () => {
+    it('returns UNAUTHORIZED if not logged in', async () => {
+      await supertest().get('/mail-services/').expect(401);
+    });
+
+    it('returns all mail services for a user', async () => {
+      const user = await UserFactory.create();
+      // create a mail service
+      const studentMailService = await mailServiceFactory.create();
+      await userSubscriptionFactory.create({
+        isSubscribed: true,
+        user: user,
+        service: studentMailService,
+      });
+
+      const resp = await supertest({ userId: user.id }).get('/mail-services/');
+      expect(resp.status).toBe(200);
+    });
+  });
```

Comment:
probably want to add some more tests, like to test updating the mail service.

Could also add a test to test if a professorMailService works